subsys/bluetooth/host/audio/iso.c

```diff
@@ -1079,6 +1082,7 @@ int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf)
 	BT_DBG("chan %p len %zu", chan, net_buf_frags_len(buf));
 
 	if (!chan->conn) {
+		BT_DBG("Not connected");
```

Comment:
Seems like BT_WARN or BT_ERR might be more appropriate? Or this is a normal/common case?

Reply:
For this I don't think we need it, as it an API function that directly returns `-ENOTCONN` as well.